src/style.css

```diff
@@ -1 +1,80 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search__form {
+  position: relative;
+  color: #3d4e61;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)